src/components/topic_components/TopicCards.jsx

```diff
@@ -0,0 +1,15 @@
+import {useState} from "react";
+import {Link} from "react-router-dom";
+
+function TopicCards(){
+    const [topics, setTopics] = useState([])
```

Comment:
what is this state doing in this component exactly? You don't seem to use `setTopics` anywhere